xml/System.Diagnostics/Activity.xml

```diff
@@ -1117,9 +1159,17 @@ This method has the advantage that no string manipulation is needed to set the I
         <ReturnType>System.Diagnostics.ActivitySource</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Get the activity source associated with this activity.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+All activities created from public constructors will have a singleton source where the source name is an empty string. Otherwise, the source will hold the object that created the activity through <xref:System.Diagnostics.ActivitySource.StartActivity>.
```

Comment:
```suggestion
All activities created from public constructors will have a singleton source where the source name is an empty string. Otherwise, the source will hold the object that created the activity through <xref:System.Diagnostics.ActivitySource.StartActivity%2A>.
```